Source/Core/DolphinQt2/DPI.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/DPI.h"
+
+#include <algorithm>
+
+#include <QGuiApplication>
+#include <QPaintDevice>
+#include <QPixmap>
+#include <QScreen>
+#include <QSize>
+#include <QString>
+
+namespace DPI
+{
+double GetScale()
+{
+  return static_cast<QGuiApplication*>(QGuiApplication::instance())->devicePixelRatio() * 1.1;
```

Comment:
`primaryScreen` sounds a little like it may return the wrong thing if dolphin is _not_ on the primary screen. Not sure if thats really the case, but I've run into some interresting issues in C# using [`Screen.PrimaryScreen`](https://msdn.microsoft.com/en-us/library/system.windows.forms.screen.primaryscreen%28v=vs.110%29.aspx) - so something to keep in mind and try out.